scripts/local-lok.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+echo "Removing existing libreofficekit"
+
+rm -rf ./src/out/Default/libreofficekit
+mkdir ./src/out/Default/libreofficekit
+
+echo "Bringing over local libreofficekit"
+
+cp -r ./../libreofficekit/libreoffice-core/instdir/* ./src/out/Default/libreofficekit/
```

Comment:
This also makes quite a few assumptions:

That the current directory is the top level directory.
That the build directory is Default.
That the project has been built at least once.

At the very least, these assumptions should be noted in a comment in the script.

Alternatively, they could be addressed by simply extending the `e` script which handles a lot of the setup for you.
See `scripts/e gen_clangd` code for an example:
https://github.com/coparse-inc/electron-libreoffice/blob/main/scripts/e#L253-L259

Reply:
> Should probably be `-rf` so that it overwrites old files.

Since I remove the directory before hand `cp -r` should always be correct. I'll do `-rf` just to be safe though

Reply:
Updated the script to be more robust. But the scripts/e SCRIPT_DIR and BUILD_DIR are actually only working if you execute them from the root of the repo. SCRIPT_DIR is always set to `pwd` since the `BASH_SOURCE[0]` fails for me at least.